azavea/climate-change-lab
# Feature/kak/switch to json export#180

## Overview

Switch from CSV chart data export to JSON export, removing CSV export. Also fixes chart export image issue in #188. Refactored a couple of service names that were no longer accurate.


## Testing Instructions

 * Click export button above chart
 * Should get chart data as JSON file

Closes #180 
Fixes #188
